install.sh

```diff
@@ -50,59 +53,64 @@ vaultConfig='
         }
     }
 '
-vault_i() { 
+vault_i() {
     docker exec $containerId vault "$@"
 }
 # First argment should be the service name. Examples are "vault", "emq"
 create_tls(){
     tlsResponse=$(vault_i write -format=json ca/issue/tls common_name="${1}.${stackname}.local" ttl=720h format=pem)
-    tlsCert=$(grep -Po '"certificate":.*?[^\\]",' <<< $tlsResponse | cut -d \" -f 4)
-    tlsKey=$(grep -Po '"private_key":.*?[^\\]",' <<< $tlsResponse | cut -d \" -f 4)
-    tlsCa=$(grep -Po '"issuing_ca":.*?[^\\]",' <<< $tlsResponse | cut -d \" -f 4)
+    tlsCert=$(grep -Eo '"certificate":.*?[^\\]",' <<< "$tlsResponse" | cut -d \" -f 4)
+    tlsKey=$(grep -Eo '"private_key":.*?[^\\]",' <<< "$tlsResponse" | cut -d \" -f 4)
+    tlsCa=$(grep -Eo '"issuing_ca":.*?[^\\]",' <<< "$tlsResponse" | cut -d \" -f 4)
```

Comment:
Put `"` around $tlsResponse to preserve newlines & change the grep flag from `-P` (Perl regex) to `-E` (expression) due to Mac's lacking the `-P` flag.